README.md

```diff
@@ -112,6 +114,14 @@ export BEE_DEBUG_API_URL=http://localhost:1635
 npm run start
 ```
 
+#### 5. Extends stamps capacity
+
+```sh
+export POSTAGE_EXTENDS_CAPACITY=true
```

Comment:
Also document `POSTAGE_USAGE_THRESHOLD` here please, if I have seen correctly it determines when the stamp is diluted

Reply:
No, it doesn't determine _when_, it determine _how much_ the stamp have been use (aka the usage level of the stamp) for it to be extended. It uses the DEFAULT_POSTAGE_USAGE_THRESHOLD variable, so it is optional. I'll specify this on the **Usage** section of the README.md

Reply:
```
filterUsableStampsExtendsCapacity(stamps: PostageBatch[], usageThreshold: number): PostageBatch[]
```

This function uses `usageThreshold`, which is made from `Number(POSTAGE_USAGE_THRESHOLD || DEFAULT_POSTAGE_USAGE_THRESHOLD)`